examples/megacli.cpp

```diff
@@ -2402,6 +2413,48 @@ void exec_find(autocomplete::ACState& s)
     }
 }
 
+bool recurse_findemptysubfoldertrees(Node* n, bool movetotrash)
+{
+    if (n->type == FILENODE) return false;
+    std::vector<Node*> emptyfolders;
```

Comment:
Please, remember the code style.
```suggestion
    std::vector<Node*> emptyFolders;
```
I'll refactorize this one.